srcpkgs/libkml/patches/NOUNCRYPT.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/kml/base/contrib/minizip/unzip.c	2015-12-21 18:23:05.000000000 +0100
++++ b/src/kml/base/contrib/minizip/unzip.c	2020-11-26 15:45:55.033835816 +0100
+@@ -42,7 +42,7 @@
+ #include <zlib.h> //RR
+ #include "unzip.h"
+ #include "iomem_simple.h"
+-#undef NOUNCRYPT
++#define NOUNCRYPT
```

Comment:
Leave a comment to why this patch is necessary.

Reply:
minizip/crypt.h was removed from the packet minizip-1.2.11_2. commit [952ac913cf](https://github.com/Nyx70/void-packages/commit/952ac913cf)
with "#define NOUNCRYPT" libkml can be built without "crypt.h"